src/agent/samples/policy/yaml/deployment/deployment-back.yaml

```diff
@@ -2,34 +2,42 @@
     spec:
       nodeSelector:
         kubernetes.io/os: linux
       runtimeClassName: kata-cc
       containers:
-        - name: azure-vote-front
-          image: "mcr.microsoft.com/azuredocs/azure-vote-front:v1"
-          env:
-            - name: REDIS
-              value: azure-vote-back
-          resources:
-            requests:
-              cpu: 100m
-              memory: 128Mi
-            limits:
-              cpu: 250m
-              memory: 256Mi
+        - name: backend
+          image: "marinerconfpodsdev.azurecr.io/hashicorp/http-echo:latest"
```

Comment:
We are now taking a dependency of this image, which is a mirror of https://github.com/hashicorp/http-echo : "a small go web server that serves the contents it was started with as an HTML page". This is just to represent a "backend". I looked for and couldn't find alternative images in https://explore.ggcr.dev/?repo=mcr.microsoft.com